solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeature.java

```diff
@@ -33,6 +33,9 @@ public void testFilterFeatureScorerOverridesScorerMethods() {
         // the FilterFeatureScorer may simply inherit Scorer's default implementation
         if (scorerClassMethod.getName().equals("twoPhaseIterator")) continue;
 
+        // the FilterFeatureScorer may simply inherit Scorer's default implementation
+        if (scorerClassMethod.getName().equals("smoothingScore")) continue;
+        
```

Comment:
This surprised me. Created https://issues.apache.org/jira/browse/SOLR-15958 and (draft) https://github.com/apache/solr/pull/567 to consider further.